https://patchew.org/QEMU/20210503161422.15028-1-dgouttegattat@incenp.org/
https://gitlab.com/qemu-project/qemu/-/issues/256
https://bugs.gentoo.org/789561

From c1155afabf1a3e093049975ba1748cf39734aa52 Mon Sep 17 00:00:00 2001
From: Damien Goutte-Gattat via <qemu-devel@nongnu.org>
Date: Mon, 3 May 2021 17:14:22 +0100
Subject: [PATCH] docs: Fix installation of man pages with Sphinx 4.x

The 4.x branch of Sphinx introduces a breaking change, as generated man
pages are now written to subdirectories corresponding to the manual
section they belong to. This results in `make install` erroring out when
attempting to install the man pages, because they are not where it
expects to find them.

This patch restores the behavior of Sphinx 3.x regarding man pages.

Signed-off-by: Damien Goutte-Gattat <dgouttegattat@incenp.org>
---
 docs/conf.py | 1 +
 1 file changed, 1 insertion(+)

--- a/docs/conf.py
+++ b/docs/conf.py
@@ -269,6 +269,7 @@
      ['Stefan Hajnoczi <stefanha@redhat.com>',
       'Masayoshi Mizuma <m.mizuma@jp.fujitsu.com>'], 1),
 ]
+man_make_section_directory = False
 
 # -- Options for Texinfo output -------------------------------------------
 
-- 
2.31.1

